IN : User Agent addition on certain streams + Cleaning

Niyasmkkv opened the pull request.

Niyasmkkv committed:
added user client for kodi

LaneSh4d0w commented on channels/in.m3u:
```diff
@@ -556,10 +556,12 @@ http://208.115.215.42/Sony_Max_HD/playlist.m3u8
 http://wagmaott.com:8080/mag00:1A:79:6A:63:7A/fYx9iEO1Lu/7906
 #EXTINF:-1 tvg-id="SonySabHD.in" tvg-country="IN" tvg-language="" tvg-logo="" group-title="",Sony Sab HD (1080p)
 http://indo51.gcdn.co/hindi-SONYSABHD/tracks-v1a1/mono.m3u8
+#EXTVLCOPT:http-user-agent=stream
```
Those need to be added after the channel info line, or before the link.

BellezaEmporium committed:
Merge branch 'master' into patch-35

BellezaEmporium committed:
Update in.m3u

BellezaEmporium committed:
Few changes + clean

Deleted certain streams that shouldn't be here (xtream-codes)
Moved the stream mentions

BellezaEmporium committed:
Merge branch 'master' into patch-35

LaneSh4d0w commented:
Made the changes myself so that you could see how it's done.

BellezaEmporium committed:
No need for "miniproxy" for Utsav Gold

Dum4G committed:
Merge branch 'master' into patch-35
